### PR TITLE
concepts: add idref

### PR DESCRIPTION
**Pull request by rerowep**
Co-Authored-by: Peter Weber <peter.weber@rero.ch>

### CODE REVIEW

File: rero_mef/api_mef.py [hunk 1]
```diff
@@ -115,47 +115,47 @@ def get_pids_with_multiple_mef(cls, record_types=[], verbose=False):
...
-                    data = hit.to_dict()
-                    mef_pid = data['pid']
-                    agent_pid = data[agent_name]['pid']
-                    pids[record_type].setdefault(agent_pid, [])
-                    pids[record_type][agent_pid].append(mef_pid)
-                    if len(pids[record_type][agent_pid]) > 1:
-                        multiple_pids[record_type][agent_pid] = \
-                            pids[record_type][agent_pid]
-                if len(pids[record_type]) < agent_class.count():
+        if record_types:
```
Comment (BadrAly) [inline rero_mef/api_mef.py:118]: I do not think you need this `if` statement.

File: rero_mef/concepts/idref/__init__.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+#
+# RERO MEF
+# Copyright (C) 2020 RERO
```
Comment (BadrAly) [inline rero_mef/concepts/idref/__init__.py:4]: `2022`, same remark applies to all files

File: rero_mef/concepts/mef/api.py [hunk 3]
```diff
@@ -92,12 +92,11 @@ def replace_refs(self):
...
         data = super().replace_refs()
         sources = []
-        for agent in ['rero']:
-            if agent in data and data[agent]:
-                sources.append(agent)
-                metadata = data[agent].get('metadata')
-                if metadata:
-                    data[agent] = metadata
+        for concept in self.entities:
+            if concept in data and data.get(concept):
```
Comment (BadrAly) [inline rero_mef/concepts/mef/api.py:96]: do you need really the `and`?

**Review by BadrAly — CHANGES_REQUESTED** [review]

File: rero_mef/api_mef.py (hunk 1 above)
Comment (rerowep) [reply rero_mef/api_mef.py:118]: We need this because record_types could be None.

**Pull request merged**